app/frontend/src/features/communities/CommunityPage/DiscussionCard.tsx

```diff
@@ -45,41 +47,26 @@ export default function DiscussionCard({
 }) {
   const classes = useStyles();
   const { data: creator } = useUser(discussion.creatorUserId);
-  //although we are only using 1 page of query, still use infinite as that
-  //is how data is stored under that query key
-  const { data: thread } = useInfiniteQuery<GetThreadRes.AsObject, GrpcError>(
-    threadKey(discussion.threadId),
-    () => service.threads.getThread(discussion.threadId),
-    {
-      enabled: !!creator,
-    }
-  );
-  const replyUserIds =
-    (thread?.pages.length ?? 0) > 0
-      ? thread!.pages[0].repliesList.map((reply) => reply.authorUserId)
-      : [];
-  const { data: replyUsers } = useUsers(replyUserIds);
 
   const date = discussion.created
     ? timestamp2Date(discussion.created)
     : undefined;
   const posted = date ? timeAgo(date, false) : "sometime";
-  const strippedText = useMemo(
-    () => stripMarkdown(discussion.content.replace("\n", " ")),
-    [discussion.content]
-  );
-  const textTruncated =
-    strippedText.length > 300
+  const truncatedContent = useMemo(() => {
+    const strippedText = stripMarkdown(discussion.content.replace("\n", " "));
+    return strippedText.length > 300
       ? strippedText.substring(0, 298) + "..."
       : strippedText;
+  }, [discussion.content]);
 
   return (
     <Card className={classNames(classes.root, className)}>
       <Link to={routeToDiscussion(discussion.discussionId, discussion.slug)}>
-        <CardActionArea>
```

Comment:
Why is this gone? I think it sets hover styling for the link.

Reply:
Oops I think you're right. I thought it was redundant since it didn't affect the button function 🙈

Reply:
`CardActionArea` renders a "button", and there doesn't seem to be a way to override it, so this also leads to the invalid interactive element nesting under `<a>` issue. I implemented the hover in CSS instead